defaults/main.yml

```diff
@@ -1,2 +1,4 @@
 ---
 spark_version: "1.2.0+cdh5.3.*"
+
+spark_env_extras: []
```

Comment:
This makes sense and don't think it's hard to adapt what I've done. I'll give it a shot and see how it works